### PR TITLE
chore: update workflow and code style

### PR DESCRIPTION
**Pull request by varl**
- ci(preview): update to latest workflow
- chore(deps-dev): update cli-style to latest
- style: update code style to latest

### CODE REVIEW

File: components/alert/src/alert-bar/icon.js [hunk 1]
```diff
@@ -31,7 +31,6 @@ StatusIcon.defaultProps = {
 }
 
 StatusIcon.propTypes = {
-    className: PropTypes.string,
```
Comment (varl) [inline components/alert/src/alert-bar/icon.js:34]: [ESLint] Remove unused property.

File: components/select/src/multi-select/multi-select.js [hunk 2]
```diff
@@ -31,11 +31,8 @@ StatusIcon.defaultProps = {
 }
 
 StatusIcon.propTypes = {
```
Comment (varl) [inline components/select/src/multi-select/multi-select.js:33]: [ESLint] Remove unused properties (className, info, warning).

File: components/select/src/single-select/single-select.js [hunk 3]
```diff
@@ -31,11 +31,8 @@ StatusIcon.defaultProps = {
 }
 
 StatusIcon.propTypes = {
```
Comment (varl) [inline components/select/src/single-select/single-select.js:33]: [ESLint] Remove unused properties (className, info, warning).

File: storybook/config/preview.js [hunk 4]
```diff
@@ -64,7 +64,6 @@ export const parameters = {
     },
     docs: {
         // Customize docs page layout (in order to rename 'Stories' section)
-        /* eslint-disable-next-line react/display-name */
```
Comment (varl) [inline storybook/config/preview.js:67]: [ESLint] Remove unused ignore directive.

**Review by Mohammer5 — APPROVED** [review]

**Pull request merged**